docs/deep-learning/lora_training.md

```diff
@@ -7,22 +7,20 @@ Small models can be fine-tuned using a single-client/single-server setup. For la
 ## Overview
 
 {% hint style="info" %}
-Refer to [this notebook](../advanced_examples/LoraMLP.ipynb) to see the tutorial about applying FHE LORA fine-tuning to a small neural network.
+Refer to [this notebook](../advanced_examples/LoraMLP.ipynb) to see the tutorial about applying FHE LoRA fine-tuning to a small neural network.
 {% endhint %}
 
-Concrete ML supports LORA, a parameter efficient fine-tuning (PEFT) approach, in the [hybrid model](../guides/hybrid-models.md) paradigm. LORA adds adapters, which contain a low number of fine-tunable weights, to the linear layers in an original model.
+Concrete ML supports LoRA, a parameter-efficient fine-tuning (PEFT) approach, in the [hybrid model](../guides/hybrid-models.md) paradigm. LoRA adds adapter layers, which contain a small number of trainable parameters, to the linear layers of a base model.
 
-In this setup, Concrete ML outsources the forward and backward passes of the model's original logic to one or more remote servers. Meanwhile, the forward and backward passes over the LORA weights, the loss computation and the weight updates are performed by the client side. As the number of LORA weights is low, this does not significantly increase the computational load for the model training client machine. For large LLMs, over 99% of the model's weights can be outsourced.
+In this setup, Concrete ML outsources the computationally intensive parts of forward and backward passes for large models to one or more remote servers. The training client machine only handles the LoRA-adapter forward/backward passes, loss computation, and adapter weight updates. Since the LoRA adapters are small, this additional computation on the client side is minimal. For large LLMs, over 99% of the model's weights can remain outsourced.
```

Comment:
in `For large LLMs, over 99% of the model's weights can remain outsourced.` 
isn't it rather: `For large LLMs, over 99% of the model's linear weights can remain outsourced.`

Reply:
Yes true we don't yet outsource embedding layers.

Reply:
Maybe specify that it's for linear layers, the reader may think, that we outsource 99% of the model's weights (including all type of layers).
I find the sentence misleading